easybuild/easyconfigs/b/BamUtil/BamUtil-libStatGen-1.0.13-foss-2015b.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Author: Adam Huffman
+# adam.huffman@crick.ac.uk
+# The Francis Crick Institute
+#
+# This is the version bundled with the libStatGen library
+
+name = 'BamUtil'
+version = '1.0.13'
+versionsuffix = 'libStatGen'
```

Comment:
we typically start a versionsuffix with `-` (and you'll need to fix the filename of the easyconfig accordingly, a `versionsuffix` goes after the `version` bit in the filename...)